web-registry/src/pages/MyEcocredits/MyEcocredits.tsx

```diff
@@ -191,6 +190,13 @@ export const MyEcocredits = (): JSX.Element => {
     setTxButtonTitle,
   });
 
+  const sxs = {
+    arrow: {
+      width: theme.spacing(6),
+      height: theme.spacing(6),
+    } as SxProps<Theme>,
+  };
```

Comment:
example of how reused `sx` props can be extracted. 

Another option would be to wrap the entire `AvailaleCreditsIcon` in `styled`